src/Components/HouseholdDataBlock/HouseholdDataBlock.js

```diff
@@ -414,12 +423,28 @@ const HouseholdDataBlock = ({ handleHouseholdDataSubmit }) => {
     }
   };
 
+  const displayHealthInsuranceQuestion = (page, hhMemberData, setHHMemberData) => {
+    return (
+      <>
+        <HealthInsuranceQ
+          hhMemberIndex={page}
+          householdMemberData={hhMemberData}
+          setHouseholdMemberData={setHHMemberData}
+          healthInsuranceErrorController={healthInsuranceErrorController}
+        />
+        <p className="household-data-q-underline"></p>
+      </>
+    );
+  };
+
   return (
     <main className="benefits-form">
       <div>
         {createQuestionHeader(page)}
         {createAgeQuestion(page)}
+        {page === 1 && displayHealthInsuranceQuestion(page, householdData, setHouseholdData)}
         {page !== 1 && createHOfHRelationQuestion()}
+        {page !== 1 && displayHealthInsuranceQuestion(page, householdData, setHouseholdData)}
```

Comment:
```suggestion
        {displayHealthInsuranceQuestion(page, householdData, setHouseholdData)}
        {page !== 1 && createHOfHRelationQuestion()}
```

Reply:
I set it up this way because the ticket specified that the health insurance question should be listed after the `createHOfHRelationQuestion` for subsequent household members.

Reply:
This should work then.
```suggestion
        {page !== 1 && createHOfHRelationQuestion()}
        {displayHealthInsuranceQuestion(page, householdData, setHouseholdData)}
```